components/Breadcrumb.vue

```diff
@@ -45,60 +45,79 @@
+import { getKeycloakRoles } from '@/utils'
 
 @Component({})
 export default class Breacrumb extends Vue {
-  get backDisabled(): boolean {
-    if ((this.$route.path === '/dashboard') || (this.$route.path === '/')) {
-      return true
+  /** Whether the back button should be disabled. */
+  get isBackDisabled(): boolean {
+    switch (this.$route.path) {
+      case '/':
+      case '/dashboard':
+      case '/dashboard/': return true
     }
     return false
   }
 
+  /** The list of breadcrumbs for the current route. */
   get breadcrumbs(): Array<any> {
-    if ((this.$route.path === '/ppr-marketing') || (this.$route.path === '/ppr-marketing/')) {
-      return [
-        {
-          disabled: false,
-          href: '/',
-          text: 'BC Registries and Online Services',
-        },
-        {
-          disabled: true,
-          href: '',
-          text: 'Personal Property Registry',
-        },
-      ]
-    }
-
-    if ((this.$route.path === '/dashboard') || (this.$route.path === '/dashboard/')) {
-      return [
-        {
-          disabled: true,
-          href: '',
-          text: 'BC Registries Dashboard',
-        },
-      ]
-    }
-
-    if (this.$route.path === '/') {
-      return [
-        {
-          disabled: true,
-          href: '',
-          text: 'BC Registries and Online Services',
-        },
-      ]
+    console.log('*** path =', this.$route.path) // TODO: remove after debugging
+
+    switch (this.$route.path) {
+      case '/ppr-marketing':
+      case '/ppr-marketing/': {
+        return [
+          {
+            disabled: false,
+            href: '/',
+            text: 'BC Registries and Online Services',
+          },
+          {
+            disabled: true,
+            href: '',
+            text: 'Personal Property Registry',
+          },
+        ]
+      }
+
+      case '/dashboard':
+      case '/dashboard/': {
+        let isStaff: boolean
+        try {
+          const keycloakRoles = getKeycloakRoles()
+          isStaff = (keycloakRoles.includes('gov_account_user') || keycloakRoles.includes('staff'))
+        } catch(e) {
+          isStaff = false
+        }
+
+        return [
+          {
+            disabled: true,
+            href: '',
+            text: isStaff ? 'Staff Dashboard' : 'BC Registries Dashboard',
+          },
+        ]
+      }
+
+      case '/': {
+        return [
+          {
+            disabled: true,
+            href: '',
+            text: 'BC Registries and Online Services',
+          },
+        ]
+      }
     }
 
     return []
   }
 
+  /** The back URL for the current route. */
   get backUrl(): string | boolean {
-    if ((this.$route.path === '/ppr-marketing') || (this.$route.path === '/ppr-marketing/')) {
-      return '/'
-    }
-    return false
+    // for now, in all cases, the back URL is the top route
+    // this can be updated later if the back URL is anything else
+    return '/'
```

Comment:
For example, a future route might be `/business-marketing`.

For others, the back button is disabled anyway so `/` doesn't do anything.